src/unity/toolkits/coreml_export/MLModel/src/BayesianProbitRegressionValidator.cpp

```diff
@@ -1,11 +1,8 @@
-//
-//  BayesianProbitRegressionValidator.cpp
-//  mlmodelspec
-//
-//  Created by Roman Holenstein on 7/18/17.
-//  Copyright © 2017 Apple. All rights reserved.
-//
-
+/* Copyright © 2017 Apple Inc. All rights reserved.
```

Comment:
All the changes under `coreml_export/MLModel` will need to go into [coremltools](https://github.com/apple/coremltools/tree/master/mlmodel) as well.